src/analysis/retail/monk/windwalker/modules/spells/FistsofFury.tsx

```diff
@@ -173,43 +154,14 @@ class FistsofFury extends Analyzer {
             Fists of Fury ticks 5 times over the duration of the channel
             <br />
             <br />
-            Note: During <SpellLink spell={TALENTS_MONK.SERENITY_TALENT} />, Fists of Fury should be
-            clipped by <SpellLink spell={SPELLS.RISING_SUN_KICK_DAMAGE} />. The graphs shown below
-            are just for information.
+            Note: During Serenity, Fists of Fury should be clipped by{' '}
```

Comment:
Remove this note I think now that serenity is gone and fof shouldn't be clipped

Reply:
Agreed, I thought I got it everywhere but good thing the formatting pointed one out I missed!